resources/conf/config.toml.template

```diff
@@ -3,6 +3,9 @@
 [adapter] # --------------------------------------------------------
 
 artifactsDirectory = "/home/wso2/artifacts"
+# Sandbox vhost for Choreo that exposes the sandbox endpoint
+sandboxVhost = "sandbox.host"
```

Comment:
Looks like this is a Router config, the same as the SystemHost config.

Reply:
I kept this here, because the usage of this value depends on a logic in the adapter. This value is used only if the API deployed contains the `sandboxEndpointChoreo` param. Unlike system host which is always there in the router.

Reply:
Shall we introduce a config as below?

[Choreo.Cloud]
sandboxVhost = "sandbox.host"

@VirajSalaka @suksw

Reply:
As far as I understood this is PR is not the final solution for the sandbox feature. Therefore, keeping the config as is, since the PR has been already merged.